kernel/uid_gid.c

```diff
@@ -1000,3 +1020,50 @@ long myst_syscall_fchown(int fd, uid_t owner, gid_t group)
 
     return ret;
 }
+
+long myst_syscall_lchown(const char* pathname, uid_t owner, gid_t group)
```

Comment:
This, `myst_syscall_fchown`, `myst_syscall_chown`, and `myst_syscall_fchownat` should be put together in a new file `chown.c`.

Reply:
Done.